### DISCUSSION TITLE
Will there be option to add more activations/seats to the license

### DISCUSSION THREAD

**Discussion by amarkulo** [spatie/ray#814] [General]
I'm just curious, but will there be a way to purchase more seats for single license? 

I'm at the moment using 3-4 different machines and would like to have the same license extend over all of them.

Thanks for the great piece of software I cannot live without :)

**Comment by olsza** [comment]
I have the same problem. I have a lifetime licence and three laptops at home, which I use two at a time, as well as a computer at work. I have to switch between licences.
I understand the need to prevent licence abuse, but perhaps there could be a solution where only two licences can be active at the same time and the rest are 'sleeping'. For example, when I use it at work, the others can be 'asleep' at home, and vice versa.
